### PR TITLE
creates react sdk for react context + hooks

### PR DESCRIPTION
**Pull request by AlwaysBCoding**
Isolates the React SDK into it's own package

### CODE REVIEW

File: packages/sdk-react/src/contexts/TurnkeyContext.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  const TurnkeyIframeContainerId = "turnkey-auth-iframe-container-id";
+
+  useEffect(() => {
+
+    (async () => {
+      if (!iframeInit.current) {
+        iframeInit.current = true;
+        const tkClient = new TurnkeyBrowserSDK(config);
+        setTurnkeyClient(tkClient);
+        const ifClient = await tkClient.iframeSign(document.getElementById(TurnkeyIframeContainerId));
```
Comment (andrewkmin) [inline packages/sdk-react/src/contexts/TurnkeyContext.tsx:38]: tiny nit: would prefer `iframeClient` here

File: packages/sdk-react/package.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  },
+  "files": [
+    "dist/",
+    "CHANGELOG.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "version": "node -p \"'export const VERSION = ' + JSON.stringify(require('./package.json').name + '@' + require('./package.json').version) + ';'\" > src/__generated__/version.ts",
```
Comment (andrewkmin) [inline packages/sdk-react/package.json:41]: I don't think we need this for this package

File: packages/sdk-react/package.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    "url": "https://turnkey.com"
+  },
+  "homepage": "https://github.com/tkhq/sdk",
+  "bugs": {
+    "url": "https://github.com/tkhq/sdk/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tkhq/sdk.git",
+    "directory": "packages/sdk-js-react"
```
Comment (andrewkmin) [inline packages/sdk-react/package.json:31]: Should rename to `packages/sdk-react` (or have everything named as `packages/sdk-js-react`)

File: packages/sdk-react/package.json (hunk 2 above)
Comment (AlwaysBCoding) [reply packages/sdk-react/package.json:41]: addressed

File: packages/sdk-react/package.json (hunk 3 above)
Comment (AlwaysBCoding) [reply packages/sdk-react/package.json:31]: addressed

File: packages/sdk-react/src/contexts/TurnkeyContext.tsx (hunk 1 above)
Comment (AlwaysBCoding) [reply packages/sdk-react/src/contexts/TurnkeyContext.tsx:38]: addressed

**Pull request merged**